packages/lib/constants.ts

```diff
@@ -164,6 +164,10 @@ export const ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE = 0;
 export const ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE_HELPER_DIALOGUE = 2;
 
 export const CALCOM_PRIVATE_API_ROUTE = process.env.CALCOM_PRIVATE_API_ROUTE || "https://goblin.cal.com";
+export const WEBSITE_PRIVACY_POLICY_URL =
+  process.env.NEXT_PUBLIC_WEBSITE_PRIVACY_POLICY_URL || "https://cal.com/privacy";
+export const WEBSITE_TERMS_URL = process.env.NEXT_PUBLIC_WEBSITE_TERMS_URL || "https://cal.com/terms";
```

Comment:
```suggestion
export const WEBSITE_PRIVACY_POLICY_URL =
  process.env.NEXT_PUBLIC_WEBSITE_PRIVACY_POLICY_URL || `${WEBSITE_URL}/privacy`;
export const WEBSITE_TERMS_URL = process.env.NEXT_PUBLIC_WEBSITE_TERMS_URL || `${WEBSITE_URL}/terms`;
```

Reply:
In the related issue Peer commented to use Cal.com's terms and conditions as fallback. See https://github.com/calcom/cal.com/issues/15373#issuecomment-2159186611

Your suggested change would lead to a 404 error on self-hosted instances (see PR see Signup page before screenshot).

Reply:
Have you check the WEBSITE_URL constants? `const WEBSITE_URL = process.env.NEXT_PUBLIC_WEBSITE_URL || "https://cal.com";` . Please change it

Reply:
Yes, I read it. In the case of my self-hosted instance `NEXT_PUBLIC_WEBSITE_URL` is set to `https://scheduling.<MY_DOMAIN>.com`. That means `WEBSITE_URL` would be exactly this and then the URLs `https://scheduling.<MY_DOMAIN>.com/terms` and `https://scheduling.<MY_DOMAIN>.com/privacy` would still lead to non-existing pages and thus produce HTTP 404 errors.

Reply:
Ooo, i see